velox/connectors/hive/HiveConfig.h

```diff
@@ -136,48 +133,6 @@ class HiveConfig {
   /// meta data together. Optimization to decrease the small IO requests
   static constexpr const char* kFilePreloadThreshold = "file-preload-threshold";
 
-  /// Maximum stripe size in orc writer.
```

Comment:
Can you update configs.rst and maybe have one section for each supported file format? Thanks!

Reply:
Will do this.

Reply:
done!